timescaledb/overview/data-model-flexibility/narrow-data-model.md

```diff
@@ -10,7 +10,8 @@ metric/tag-set combination
 In this model, each metric/tag-set combination is considered an individual
 "time series" containing a sequence of time/value pairs.
 
-Using our example above, this approach would result in 9 different "time series", each of which is defined by a unique set of tags.
+Using our example above, this approach would result in 9 different "time
```

Comment:
```suggestion
In this example, the approach would result in 9 different time
```

I like to try and avoid directional language, because things move. It's also not great for a11y generally (screen readers, etc, don't offer a good concept for "above").